docs/book/filters/to-e164.md

```diff
@@ -0,0 +1,66 @@
+# ToE164
+
+The `ToE164` filter can be used to format phone numbers to [E.164 format](https://en.wikipedia.org/wiki/E.164).
+
+## Basic Usage
+
+```php
+use Laminas\I18n\CountryCode;
+use Laminas\I18n\PhoneNumber\Filter\ToE164;
+
+$country = CountryCode::fromString('GB');
+$filter  = new ToE164($country);
+
+echo $filter->filter('01234 567 890'); // "+441234567890"
+```
+
+Regardless of the configured default country code, the `ToE164` filter will still format a recognisable number for any region:
+
+```php
+use Laminas\I18n\CountryCode;
+use Laminas\I18n\PhoneNumber\Filter\ToE164;
+
+$country = CountryCode::fromString('SE');
+$filter  = new ToE164($country);
+
+echo $filter->filter('44 (0) 1234 567 890'); // "+441234567890"
+```
+
+## Changing the default country code
+
+Given an already instantiated filter instance, the country code can be changed in 2 ways:
+
+- Via the `setCountryCode` method
+- Via the `setOptions` method
+
+The `setCountryCode` method accepts a `CountryCode` value object, or a 2-letter [ISO 3166 country code](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) string:
+
+```php
+use Laminas\I18n\CountryCode;
+use Laminas\I18n\PhoneNumber\Filter\ToE164;
+
+$country = CountryCode::fromString('GB');
+$filter  = new ToE164($country);
+
+$filter->setCountryCode('FR');
+// …or
+$filter->setCountryCode(CountryCode::fromString('DE'));
+```
+
+The `setOptions` method accepts an array or iterable in the following format:
+
+```php
+$options = [
+    'country-code' => 'US',
+];
+
+$filter->setOptions($options);
+```
+
+## Filtering invalid data
```

Comment:
For headlines the Chicago Manual of Style capitalization rules must be used:

1. Capitalize the first and the last word.
1. Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
1. Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.
1. Lowercase the ‘to’ in an infinitive (I want to play guitar).

The online tool [capitalizemytitle.com](https://capitalizemytitle.com/style/Chicago/) can help to capitalize and case convert the headlines.

```suggestion
## Filtering Invalid Data
```